### PR TITLE
[CLEANUP] Autoformat the code

### PR DESCRIPTION
**Pull request by oliverklee**
This is the v8.x backport of #748.

### CODE REVIEW

File: tests/CSSList/AtRuleBlockListTest.php [hunk 1]
```diff
@@ -43,7 +44,8 @@ public static function provideSyntacticlyCorrectAtRule()
                         display: flex;
                     }
                 }
-            '],
+            ',
+            ],
```
Comment (JakeQZ) [inline tests/CSSList/AtRuleBlockListTest.php:48]: The leading and trailing whitespace is arbitrary, and for readability.  So the string content and closing quote should now be futher indented.

File: tests/Value/ValueTest.php [hunk 2]
```diff
@@ -39,7 +39,7 @@ public function parsesArithmeticInFunctions($operator)
     {
         $subject = Value::parseValue(new ParserState('max(300px, 50vh ' . $operator . ' 10px);', Settings::create()));
 
-        self::assertSame('max(300px,50vh ' . $operator . ' 10px)', (string) $subject);
+        self::assertSame('max(300px,50vh ' . $operator . ' 10px)', (string)$subject);
```
Comment (JakeQZ) [inline tests/Value/ValueTest.php:42]: This surely contradicts PHP-Fixer, which added spaces after casts.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
I'm sure PHP-Fixer added the spaces after cast operators (somewhat against my taste).  So we shouldn't be removing them again.

There's a string with arbitrary whitespace that needs manual post-reformatting to make it readable again.

Shouldn't the autoformat be done first on `main`?

**Comment by oliverklee** [comment]
> Shouldn't the autoformat be done first on `main`?

Yes: #748 

I'll mark this PR here as draft for now and do the corresponding changes there.

File: tests/Value/ValueTest.php (hunk 2 above)
Comment (oliverklee) [reply tests/Value/ValueTest.php]: Done.

File: tests/CSSList/AtRuleBlockListTest.php (hunk 1 above)
Comment (oliverklee) [reply tests/CSSList/AtRuleBlockListTest.php]: Done.

**Review by JakeQZ — APPROVED** [review]

**Pull request merged**